Classes/common/CDTReplicator/CDTReplicator.h

```diff
@@ -16,6 +16,7 @@
 #import <Foundation/Foundation.h>
 
 #import "CDTReplicatorDelegate.h"
+#import "NSURLSessionConfigurationDelegate.h"
```

Comment:
Oh I see, that is very confusing and it will be for our library consumers and apple reserve the two letter prefixes for themselves, so we should name it something with a CDT prefix, I think perhaps, `CDTNSURLSessionConfigurationDelegate.h`